manifests/init.pp

```diff
@@ -64,14 +66,16 @@
   $package_name            = $nsclient::params::package_name,
   $download_destination    = $nsclient::params::download_destination,
   $config_template         = $nsclient::params::config_template,
-  $install_path            = $nsclient::params::install_path
+  $install_path            = $nsclient::params::install_path,
+  $proxy_url               = $nsclient::params::proxy_url
```

Comment:
As in the other PR, please add a datatype and remove the validate_string() function.